### ISSUE TITLE
Google Play: publish a new build of the android app with API 26

### ISSUE THREAD

**Issue by mpl** [perkeep/perkeep#1222]
Now that https://perkeep-review.googlesource.com/c/perkeep/+/18666 has landed.

**Comment by cliveb** [comment]
In light of this CL. Is Google Play the best place to pull the latest
perkeep android app?
https://play.google.com/store/apps/details?id=org.camlistore&hl=en_US

**Comment by cliveb** [comment]
I tried installing linked 2018 apk. Alas, Android package installer says this apk is corrupt. My phone is Nexus 6 Android 7.1.1. https://github.com/perkeep/perkeep/issues/1049#issuecomment-370060518

**Comment by mpl** [comment]
yes, Google Play is the best place to install from at the moment, so that is where I advise you to install from. If we don't update it on Google Play soon, I'll just upload an apk to our cloud bucket as usual.

**Comment by mpl** [comment]
the apk linked at https://github.com/perkeep/perkeep/issues/1049#issuecomment-370060518 was just a debug version, please do not use that.

**Comment by NiklasMerz** [comment]
Looks like the app was recently updated. https://play.google.com/store/apps/details?id=org.camlistore
